tools/uniform_sampling.cpp

```diff
@@ -49,13 +53,20 @@ using namespace pcl::console;
 
 double default_radius = 0.01;
 
-Eigen::Vector4f    translation;
-Eigen::Quaternionf orientation;
+/** \brief File type for saving and loading files. */
+typedef enum FileType
```

Comment:
In C++ you don't need to `typedef` enumerations. You can safely remove `typedef` and the second `FileType`.